kythe/go/indexer/flags.bzl

```diff
@@ -1,15 +1,25 @@
 """Starlark library to build FlagConstructors textproto files"""
 
-def flag_constructor(func_name, pkg_path, name_arg_position, description_arg_position):
-    """Produces a text proto encoding of a kythe.proto.FlagConstructor.  Meant
-    to be used as a custom_flags value.
+def flag_constructor(func_name, pkg_path, name_arg_position, description_arg_position, var_arg_position = None):
+    """Produces a text proto encoding of a kythe.proto.FlagConstructor.
+
+    Meant to be used as a custom_flags value.
 
     Args:
       func_name: the name of the flag constructing function
       pkg_path: the package path of the flag constructing function
       name_arg_position: the 0-based position of the name string argument
       description_arg_position: the 0-based position of the description string argument
+      var_arg_position: the 0-based position of the variable pointer argument
     """
+    if var_arg_position:
```

Comment:
This will omit the `var_arg_position` when it's == 0? Is that intentional?

You can probably use [skylib types.is_none](https://docs-legacy.aspect.build/bazelbuild/bazel-skylib/1.0.3/docs/types.html#types.is_none)

Reply:
Does `v != None` not work?

Reply:
Seems to work